environment_set_up/ManualConfig-HPC.md

```diff
@@ -0,0 +1,62 @@
+# Getting Started with On-Prem HPC
+
+This tutorial covers how to set up and use the Pangeo environment/stack
+on  USGS High Performance Computing (HPC) systems:
+
+1) Creating a custom `hytest` conda environment
+2) Configuring [Jupyter](https://jupyter.org/) server
+3) Launching a Jupyter server
+4) Connecting to your Jupyter server and the [Dask](https://dask.pydata.org/)
+   dashboard from your personal computer
+
+This document assumes that you already have an access to
+`denali` or `tallgrass` and are comfortable using the command line.
+
+## Set up a `hytest` Conda Environment
+
+This is the same manual process as describe in the
+[Quick-Start for HPC](./QuickStart-HPC.md#1b-set-up-a-hytest-conda-environment)
+to set up a `hytest` conda environment.
+
+## Configure Jupyter
+
+Jupyter notebook servers include a password for security. This will control who
+can use your personal jupyter server on the HPC. First we
+generate the Jupyter config file then set the password:
+
+```text
+jupyter server --generate-config
+jupyter server password
+```
+
+This creates the file `~/.jupyter/jupyter_server_config.py`.
+
+Finally, we configure dask's dashboard to forward through
+Jupyter.  Add the following line to your `~/.bashrc` file:
+
+```sh
+export DASK_DISTRIBUTED__DASHBOARD__LINK="/proxy/8787/status"
+```
+
+## Start a Jupyter Notebook Server
+
+Now that we have Jupyter configured, we can start a notebook server on our interactive compute node. Use the [script provided](start_jupyter.sh) to do this.
+
+```text
+> salloc -A account_name -t 02:00:00 -N 1 srun --pty bash
+> cd $HOME/hytest/
+> ./environment_set_up/start_jupyter.sh
+```
+
+Follow the steps printed out by the script to get connected. Note that after you login using the `ssh` command in Step 2, the cursor will hang.  This is expected and you can minimize the window.
```

Comment:
I don't see an `ssh` command in step 2 (start a jupyter notebook server)

Reply:
This script and example are taken from 
https://github.com/USGS-python/hytest-development/blob/main/hpc.md

And because I'm not yet on the HPC, I can't test.  So I might need to call in some help on that. 

I think that it would be useful to rename 'Step' in the script output so that it isn't confusing with the steps in the ManualConfig doc.